Packs/RecordedFuture/Integrations/RecordedFuturePlaybookAlerts/RecordedFuturePlaybookAlerts.py

```diff
@@ -0,0 +1,280 @@
+            url_suffix='/v2/playbook_alert/lookup', demisto_args=parsed_args
+        )
+
+    def update_playbook_alerts(self) -> Dict[str, Any]:
+        parsed_args = demisto.args()
+        if ids := parsed_args.get('alert_ids'):
+            parsed_args["alert_ids"] = ids.split(",")
+        return self._call(
+            url_suffix='/v2/playbook_alert/update', demisto_args=parsed_args
+        )
+
+    def search_playbook_alerts(self) -> Dict[str, Any]:
+        parsed_args = demisto.args()
+        if categories := parsed_args.get('category'):
+            parsed_args["category"] = categories.split(",")
+        if statuses := parsed_args.get('playbook_alert_status'):
+            parsed_args["playbook_alert_status"] = statuses.split(",")
+        return self._call(
+            url_suffix='/v2/playbook_alert/search', demisto_args=parsed_args
+        )
+
+
+# === === === === === === === === === === === === === === ===
+# === === === === === === ACTIONS === === === === === === ===
+# === === === === === === === === === === === === === === ===
+
+
+class Actions:
+    def __init__(self, rf_client: Client):
+        self.client = rf_client
+
+    def _process_result_actions(
+        self, response: Union[dict, CommandResults]
+    ) -> List[CommandResults]:
+
+        if isinstance(response, CommandResults):
+            # Case when we got 404 on response, and it was processed in self.client._call() method.
+            return [response]
+        elif not isinstance(response, dict):
+            # In case API returned a str - we don't want to call "response.get()" on a str object.
+            return None  # type: ignore
+
+        result_actions: Union[List[dict], None] = response.get('result_actions')
+
+        if not result_actions:
+            return None  # type: ignore
+
+        command_results: List[CommandResults] = list()
+        for action in result_actions:
+            if 'CommandResults' in action:
+                command_results.append(CommandResults(**action['CommandResults']))
+
+        return command_results
+
+    def fetch_incidents(self) -> None:
+
+        response = self.client.fetch_incidents()
+
+        if isinstance(response, CommandResults):
+            # 404 case.
+            return
+
+        for _key, _val in response.items():
+            if _key == 'demisto_last_run':
+                demisto.setLastRun(_val)
+            if _key == 'incidents':
+                for incident in _val:
+                    attachments = list()
+                    incident_json = json.loads(incident.get("rawJSON", "{}"))
+                    if incident_json.get("panel_evidence_summary", {}).get(
+                        "screenshots"
+                    ):
+                        for screenshot_data in incident_json["panel_evidence_summary"][
+                            "screenshots"
+                        ]:
+                            file_name = f'{screenshot_data.get("image_id").replace("img:","")}.png'
```

Comment:
```suggestion
                            file_name = f'{screenshot_data.get("image_id", "").replace("img:","")}.png'
```
In the case where no screenshots, the replace method won't fail

Reply:
done